### DISCUSSION TITLE
Thoughts on adding GLAS\GFM TRE support.

### DISCUSSION THREAD

**Discussion by linker2424** [mdaus/nitro#359] [Ideas]
Just was wondering if anyone had any thoughts on adding GLAS\GFM TRE support?

**Comment by bradh** [comment]
Parsing the TREs isn't that hard. However its important to understand what you're trying to do. 

If you don't have the detailed understanding of geopositioning, then maybe this isn't a good idea. 

Maybe you do, but that isn't coming through in your questions. Sorry if that seems harsh - perhaps adding more context to your questions might help: what you've already looked at, what you've tried, what happened, and what you expected to happen instead.
